cmd/cerbos/repl/internal/repl.go

```diff
@@ -46,19 +53,25 @@ var (
 )
 
 const (
-	commentPrefix   = '#'
-	directivePrefix = ':'
-	prompt          = "-> "
-	secondaryPrompt = "> "
+	commentPrefix        = '#'
+	directivePrefix      = ':'
+	prompt               = "-> "
+	rulePrefix           = "#"
+	secondaryPrompt      = "> "
+	variablesShortPrefix = "V."
+	variablesPrefix      = "variables."
 )
 
 type REPL struct {
-	vars     variables
-	decls    map[string]*exprpb.Decl
-	reader   *liner.State
-	parser   *participle.Parser
-	output   Output
-	toRefVal func(any) ref.Val
+	output     Output
+	vars       variables
+	policyVars map[string]string
```

Comment:
There's a disconnect between the "REPL vars" and the vars from the policy here. You can see this when you type `:vars`:

```
variables = 0

variables.pending_approval = "PENDING_APPROVAL"

variables.principal_location = ""
```

We should add the policy variables to REPL variables when a policy is loaded and remove them when another policy is loaded.